src/_lib/sass/src/_toasts.scss

```diff
@@ -10,16 +10,14 @@
 .toast {
     display: flex;
     flex-direction: row;
-    align-items: baseline;
+    align-items: flex-start;
     position: relative;
     margin: 8px auto;
-    padding: 8px 24px;
+    padding: 16px 24px;
     max-width: 500px;
     background: get-color("gray", light);
```

Comment:
Not sure if this neutral bg colour ever actually shown, as from what I've seen toast is always either a success or an error, but it should probably be changed to either $level2 or $level3.